src/viewProviders/schemas.ts

```diff
@@ -144,7 +144,9 @@ export class SchemasViewProvider
   totalItemCount: number = 0;
 
   private static instance: SchemasViewProvider | null = null;
-  private constructor() {
+
+  // Public only for testing purposes.
+  constructor() {
```

Comment:
Why do we need this to be public? In other tests we could reuse `<singleton>.getInstance()` and reset the instances in beforeEach/afterEach with `<singleton>["instance"] = null`